### ISSUE TITLE
[4.0] Icon- TO fa incomplete

### ISSUE THREAD

**Issue by brianteeman** [joomla/joomla-cms#30191]
### Steps to reproduce the issue
According to the information in the descritpion of **Icon- TO fa ( part 3 )** #28075 the icon for the Save-Group button should have been changed from `icon-save` to `fas fa-save` 

There may be others I have not checked

### Actual result

![image](https://user-images.githubusercontent.com/1296369/88456915-f9785900-ce79-11ea-9401-612b8d2a357b.png)

**Comment by richard67** [comment]
Ping @N6REJ .

**Comment by infograf768** [comment]
Is'nt it simply
```
diff --git a/layouts/joomla/toolbar/iconclass.php b/layouts/joomla/toolbar/iconclass.php
index 6ed910c..a42fa6a 100644
--- a/layouts/joomla/toolbar/iconclass.php
+++ b/layouts/joomla/toolbar/iconclass.php
@@ -36,4 +36,8 @@
 	$icon = 'fas fa-save';
 }
+elseif ($icon === 'save')
+{
+	$icon = 'fas fa-save';
+}
 elseif ($icon === 'mail')
 {

```

**Comment by infograf768** [comment]
maybe also to add in `/layouts/joomla/button/iconclass.php`

**Comment by N6REJ** [comment]
i'll double check it but I think your right it SHOULD be button/iconclass.php @infograf768.. have it done in next 24hrs.. no sleep yet

**Comment by infograf768** [comment]
in both files. tested already ok the one in the diff above

**Comment by N6REJ** [comment]
closed pr #30204
